spec/factories/user_factory.rb

```diff
@@ -40,6 +40,27 @@
     end
   end
 
+  factory :prescription_user, class: 'User' do
```

Comment:
Just FYI this does not represent what a real "prescription user" would look like.  Rx requires LOA3, so fields like User.first_name, User.last_name, etc. would never be null.